boards/arm/da14695_dk_usb/da14695_dk_usb-pinctrl.dtsi

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2022 Renesas Electronics Corporation and/or its affiliates
```

Comment:
2023?

also any reason why there are 2 different copyrights lines used in all files?

Reply:
This was fixed. All files now have the same copyright line and the right year.